dist/self-service-reasons/1.0.2/cancellation-reasons_en.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "cancelForm",
+  "locale": "en",
+  "groups": [
+    {
+      "id": "cancellation",
+      "name": "Cancellation",
+      "description": {
+        "title": "",
+        "disclaimers": [
+          "The current order will be cancelled",
```

Comment:
Prefer "Cancel" in its US form since VTEX tends to favor US english instead of uk english.